controls/radchartview/features/behaviors/panzoom.md

```diff
@@ -10,85 +10,201 @@ position: 0
 
 # Scroll and Zoom 
 
+__RadChartView__ provides zoom and scroll interactivity via the __ChartPanAndZoomBehavior__ class. It is very simple to use and allows users to zoom in the chart plot area when are there is a dense area of data points that can not be seen clearly at the normal chart scale. The scroll functionality allows moving the visible area of the chart. 

```

Comment:
when are there is a dense area of data points -> when there is a dense area of data points